ui-v2/app/styles/components/tag-list/layout.scss

```diff
@@ -11,10 +14,11 @@
 }
 %tag-list dd {
   display: inline-flex;
+  flex-wrap: wrap;
   padding-left: 0px;
 }
 %tag-list dd > *:not(:last-child)::after {
-  content: ', ';
-  white-space: pre;
+  content: ',';
+  margin-right: 0.3em;
```

Comment:
I noticed that the tag-listing wasn't wrapping properly when not in listings, this fixes that up.